src/api/registration.js

```diff
@@ -0,0 +1,29 @@
+import axios from "axios"
+
+const submitRegistration = (values, store, id) => {
+  axios
+      .get("https://mocki.io/v1/6e70ca5e-cb79-4b2f-8c99-8b99b08eb542")
+      .then((result) => {id = result.data.length; console.log(id)})
```

Comment:
Why you're writing it in one line? It's hard to read such code. 
Also remove console please